src/Raven.Studio/wwwroot/Content/css/pages/traffic-watch.less

```diff
@@ -18,4 +18,14 @@
     #trafficFilter {
         width: 220px;
     }
+    
+    .dropdown {
+        .dropdown-menu.stats-menu {
+            width: 850px;
```

Comment:
can we avoid hardcoding width? Maybe use class=row, class=col-* instead?

Reply:
refactored